tests/models/t5/test_modeling_flax_t5.py

```diff
@@ -244,6 +244,30 @@ def test_model(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs()
         self.model_tester.create_and_check_model(*config_and_inputs)
 
+    def test_gradient_checkpointing(self):
```

Comment:
Great that this test passes in the standalone test class! In the final version of the PR #17843, the `test_gradient_checkpointing` test was moved to `test_modeling_flax_common`:
https://github.com/huggingface/transformers/blob/ab2006e3d6db88654526a4169e65d4bfc52da2e3/tests/test_modeling_flax_common.py#L1102
So this test will be run automatically for all modules that have the `.enable_gradient_checkpointing()` method implemented (in this case FlaxT5 and FlaxLongT5). This means we can remove this test from the `test_modeling_flax_t5` file!

Reply:
ah good to know.  I've removed it now.

Reply:
Brill, thanks! PR is good to go once we get the green light from @patil-suraj!